### PR TITLE
Make dots orange

### PR DESCRIPTION
**Pull request by voltrevo**

### CODE REVIEW

**Review by leahciMic — APPROVED** [review]
Looks good to me

File: index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ var placeDot = function(dot, opt) {
 
 var Dot = function(opt) {
   var dot = document.createElement('div');
-  dot.style.backgroundColor = opt.color || 'blue';
+  dot.style.backgroundColor = opt.color || 'orange';
```
Comment (leahciMic) [inline index.js]: I changed my mind, blue is fine.

**Review by leahciMic — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (leahciMic) [inline index.js]: actually fuck it green

**Review by leahciMic — APPROVED** [review]

File: index.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ var placeDot = function(dot, opt) {
...
   dot.style.left = '0px';
   dot.style.top = '0px';
```
Comment (leahciMic) [inline index.js:18]: this stuff belongs in the stylesheet

**Review by leahciMic — CHANGES_REQUESTED** [review]

File: index.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ var placeDot = function(dot, opt) {
 
 var Dot = function(opt) {
   var dot = document.createElement('div');
```
Comment (leahciMic) [inline index.js]: blah

Comment (leahciMic) [inline index.js]: this one is good now

Comment (leahciMic) [inline index.js:15]: blah

File: index.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ var placeDot = function(dot, opt) {
...
-  dot.style.backgroundColor = opt.color || 'blue';
+  dot.style.backgroundColor = opt.color || 'orange';
   dot.style.left = '0px';
```
Comment (leahciMic) [inline index.js:17]: sadfsdaf

**Review by leahciMic — CHANGES_REQUESTED** [review]

**Comment by voltrevo** [comment]
I think this PR has descended into madness, closing :trollface:

**Pull request closed without merging**